codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/JsonRpcProtocolGenerator.java

```diff
@@ -63,10 +63,19 @@ protected void serializeInputDocument(GenerationContext context, OperationShape
         GoWriter writer = context.getWriter();
         StructureShape input = ProtocolUtils.expectInput(context.getModel(), operation);
         String functionName = ProtocolGenerator.getDocumentSerializerFunctionName(input, getProtocolName());
-        writer.addUseImports(SmithyGoDependency.SMITHY_JSON);
 
-        // If there are no members then there's nothing to serialize
+        // Stub synthetic clone inputs mean there never was an input modeled, so nothing should be serialized.
+        if (CodegenUtils.isStubSyntheticClone(ProtocolUtils.expectInput(context.getModel(), operation))) {
+            return;
```

Comment:
If input shape is not modeled, we don't need to send the `{}` ? This seems inconsistent. I would imagine we always send an empty json object regardless, and leave it up to service to ignore it?

Reply:
This is not clear. I'm reaching out to the Smithy team to clarify this.

Reply:
Have verified that empty JSON object `{}`  should be serialized for all empty modeled input. Its not clear if this should be done for unmodeled input. Will research that separately.